src/ODB2/CMakeLists.txt

```diff
@@ -4,6 +4,14 @@ list( APPEND programs
   var_convert.py
   DESTINATION bin)
```

Comment:
Should get rid of the "DESTINATION bin" keywords, since they are extraneous.

Reply:
@srherbener , maybe I misunderstood, but when I tried removing the "DESTINATION bin" line I got an error when I executed the "ecbuild" command for the build directory. Restoring the removed line caused the error to go away. And all the other comparable "CMakeLists.txt" files in the repo also have the "DESTINATION bin" line in that place. Are you sure these words are extraneous?

Reply:
Sorry, I meant only on the list ( APPEND programs .... ) command, you need "DESTINATION bin" on the install (...) command. Check out the CMakeLists.txt file in src/marine for example.

Reply:
Ah. I did misunderstand. Sorry about that. I have committed the requested change now.